generator/.pulp-cache/build-stamp

```diff
@@ -0,0 +1 @@
+7b2482593750e1d83b2fb300c2ab41a9
```

Comment:
These are link files to files in `generator/src`. Either `pulp` or `psc` doesn't want to follow directory links but will follow individual file links.